### DISCUSSION TITLE
Text Shape tool is now Emboss Tool?

### DISCUSSION THREAD

**Discussion by nickr1957** [OrcaSlicer/OrcaSlicer#4096] [Q&A]
I maybe missed some announcement but if I open an existing 3mf that includes a text part in an object, when I select the text part the Text Shape tool becomes an Emboss tool and I cannot edit the existing text part.

If I open the same .3mf in Bambu Studio, I can select the text part and the Text Shape tool remains a Text Shape tool and I can edit the text.

Have I missed documentation of a change ?

**Comment by netweaver1970** [comment]
Yes, emboss tool came from PrusSlicer. And there was a message saying that the pre-change project file texts couldn't be edited anymore after the change. That was the only drawback of moving to the way more featured PS way working

**Comment by nickr1957** [comment]
Ah thanks! I did miss that one.
NP, can always do it in Bambu.
The Emboss tool looks far more capable, pity it can't be used retrospectively.

Thanks again!



Sent from my Galaxy



-------- Original message --------
From: netweaver1970 ***@***.***>
Date: 15/02/2024 16:48 (GMT+00:00)
To: SoftFever/OrcaSlicer ***@***.***>
Cc: Nick Ramsey ***@***.***>, Author ***@***.***>
Subject: Re: [SoftFever/OrcaSlicer] Text Shape tool is now Emboss Tool? (Discussion #4096)


Yes, emboss tool came from PrusSlicer. And there was a message saying that the pre-change project file texts couldn't be edited anymore after the change. That was the only drawback of moving to the way more featured PS way working

**Comment by DeltaPheonix-Dev** [comment]
I have an issue with the latest Stable Windows version whereby, any attempt to use the Text tool crashes the application. Is there a work around or patch that I may use or is it possible the 3mf file type that the current version conflicts with?
